### PR TITLE
[PDMO-58] retrieve deleted identifiers

### PR DESCRIPTION
**Pull request by Ewanjiru**
# JIRA TICKET NAME:

PDMO-58 [Retrieve Deleted identifiers](https://issues.openmrs.org/browse/PDMO-58)

# SUMMARY:

Previously one could not retrieve a voided identifier due to a bug in the web service.
With the fix in the web-service, these identifiers can now be retrieved.

### CODE REVIEW

**Comment by cnagadya** [comment]
Looks great, save for the conflicts in the **obsadmin.css**file with the master branch. Be sure to check them out

**Comment by Ewanjiru** [comment]
@cnagadya  well noted. Thanks.

**Review by CeciliaWanjiku — COMMENTED** [review]
Looks good

**Comment by jacksono** [comment]
Looks good, I see that in some cases u spell 'preferred', and others u spell 'preffered'. I think the former is the correct spelling

File: app/js/components/identifiers/identifiers.jsx [hunk 1]
```diff
@@ -281,15 +325,22 @@ export default class Identifiers extends React.Component {
...
                         }
-                        <center><h4>Identifier {id.identifier}</h4></center>
                       </div>
                       <div className="card-body">
-                        <div className={this.state.editErrors.uuid === id.uuid ? editErrorClass : ''}>
+                        <div
+                          className={this.state.editErrors.uuid === id.uuid ?
+                            editErrorClass : ''}>
                           <h6><b>Identifier</b></h6>
                           <input
```
Comment (jacksono) [inline app/js/components/identifiers/identifiers.jsx]: Could this className fit on oneline?

Comment (Ewanjiru) [reply app/js/components/identifiers/identifiers.jsx]: @jacksono  Working on it.

**Comment by Ewanjiru** [comment]
All Done.

**Pull request merged**